lib/cocoapods/installer.rb

```diff
@@ -514,7 +514,7 @@ def create_pod_installer(pod_name)
         raise StandardError, message
       end
 
-      pod_installer = PodSourceInstaller.new(sandbox, specs_by_platform, :can_cache => installation_options.clean?)
+      pod_installer = PodSourceInstaller.new(sandbox, podfile, specs_by_platform, :can_cache => installation_options.clean?)
```

Comment:
should this be passed in the podfile dependency cache instead of the raw podfile?

Reply:
I think there is none at that time. Analyzer constructs one by using the `podfile`:
```ruby
@podfile_dependency_cache = PodfileDependencyCache.from_podfile(podfile)
```

I could do the same but I do not see strong benefit from it as groups dependencies by target definition.